src/Raven.Client/Documents/Session/AsyncDocumentSession.CountersFor.cs

```diff
@@ -0,0 +1,25 @@
+//-----------------------------------------------------------------------
+// <copyright file="AsyncDocumentSession.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Raven.Client.Documents.Session
+{
+    /// <summary>
+    /// Implements Unit of Work for accessing the RavenDB server
+    /// </summary>
+    public partial class AsyncDocumentSession
+    {
+        public AsyncSessionDocumentCounters CountersFor(string documentId)
```

Comment:
Just in case I would go with Interface here.

Reply:
we have one already https://github.com/ravendb/ravendb/pull/6549/files#diff-eefe503b561793713fc89e176ec41b71

Reply:
I mean as a return type of this method.

Reply:
done